edoweb/edoweb.module

```diff
@@ -1666,9 +1666,25 @@ function edoweb_basic_view($entity, $view_mode = 'default') {
   
   }
   else {
+	$prefix = "<div class=\"edoweb entity $view_mode\" $attr_bundle $attr_rdf_resource $attr_rdf_typeof>"
+  			. '<div class="compact-header">' . _full_title($entity->label(), $entity);
+ 	$api = new EdowebAPIClient();
+	if (!$conf = $api->getCrawlerConfiguration($entity)) {
+		$conf = array();
+	}
+	if( @$conf['invalidUrl'] != null && @$conf['invalidUrl'] == true ) {
+        	$url = $entity->url() . '/crawler';
+		if( @$conf['urlNew'] != null ) {
+			$img = '<img src="/public/images/icon_lastwagen.jpeg" title="Die Site ist umgezogen nach ' . @$conf['urlNew'] . ' (Bitte bestätigen!)" />';
+	 		$prefix .=  '&nbsp;<span><a href="'.$url.'">'.$img.'</a></span>';
+		} else {
+			$img = '<img src="/public/images/icon_lastwagen.jpeg" title="Die Site ist umgezogen !" />';
+	 		$prefix .=  '&nbsp;<span><a href="'.$url.'">'.$img.'</a></span>&nbsp;<span class="octicon octicon-alert" title="Website unbekannt verzogen !"></span>';
+		}
+	}
+	$prefix .= '</div>';
```

Comment:
@jschnasse : In den Zeilen 1669 - 1687 ist das Grüne herauszunehmen und das Rote zurückzuholen. Dadurch werden alle Referenzen auf das Lastwagensymbol entfernt,

Reply:
ok, kannst Du so machen.